docs/docs/events/index.md

```diff
@@ -0,0 +1,71 @@
+# Events
+
+Events refer to the activity in a system that tracee monitors. There are two main types of events, built-in events that are part of Tracee and custom events, which references the way users define other events that Tracee should monitor. 
+
+As part of built-in events, there are six types of events:
+
+* syscalls 
+* network 
+* security 
+* lsm 
+* containers 
+* misc
+
+This section documents all of the different events that Tracee exposes.
+
+## Defining events
+
+Tracing the `execve` events in a [policy](../policies/index.md):
+
+```
+apiVersion: tracee.aquasec.com/v1beta1
+kind: Policy
+metadata:
+	name: sample-policy
+	annotations:
+		description: traces execve events
+spec:
+	scope:
+	  - global
+	rules:
+	  - event: execve
+```
+
+If no event is passed with [filters] or [policies], tracee will start with a set of default events.
```

Comment:
Not sure it's correct. I think it should be "if no policy is defined, tracee will start with.."